WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -308,6 +308,7 @@ protected void onCreate(Bundle savedInstanceState) {
                     postFormat = SiteSettingsInterface.getDefaultFormat(WordPress.getContext());
                 }
                 mPost = mPostStore.instantiatePostModel(mSite, mIsPage, categories, postFormat);
+                mPost.setStatus(PostStatus.PUBLISHED.toString());
```

Comment:
Our default status has always been `PUBLISHED` - it's the status of the post if you write a new post and hit 'Publish' without changing anything. `DRAFT` is actually a special case. `Publish` is also what you would see as the default setting if you opened the post settings screen.